ee/modules/650-runtime-audit-engine/images/rules-loader/hooks/rules/hook.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+
+from os import remove, walk
+
+from shell_operator import hook
+from yaml import dump
+
+_FALCO_RULES_DIR = '/etc/falco/rules.d'
+
+
+# Converts FalcoAuditRules CRD format to the native Falco rules
+def convert_spec(spec: dict) -> list:
+    result = []
+
+    required_engine_version = spec.get("requiredEngineVersion")
+    if required_engine_version is not None:
+        result.append({
+            "required_engine_version": required_engine_version,
+        })
+
+    required_k8saudit_plugin_version = spec.get("requiredK8sAuditPluginVersion")
+    if required_k8saudit_plugin_version is not None:
+        result.append({
+            "required_plugin_versions": [
+                {
+                    "name": "k8saudit",
+                    "version": required_k8saudit_plugin_version,
+                },
+            ],
+        })
+
+    for item in spec["rules"]:
+        if item.get("rule") is not None:
```

Comment:
```suggestion
        if "rule" in item:
```

etc.

Reply:
This is due to avoid the edge case:
```python
rule: null
```
On the other hand, we can just say that CRD validates this, but I'm not sure this is better to rely on open API, because there are too many moving parts.

Reply:
Agreed, your variant is more robust. 

On the other hand, I'd like to know, what we receive in nullable fields depending on the schema, i.e. if they are nulls or they are missing.

Reply:
Let's leave a comment about the expression and nullable value, to describe why it is a little non-idiomatic for Python.

Reply:
Added